File: extmod/modtrezorcrypto/ssss.c
#include "ssss.h"

bool ssss_split(const bignum256 *secret, int m, int n, bignum256 *shares)
{
	if (m < 1 || n < 1 || m > 15 || n > 15 || m > n) {
		return false;
	}
	return true;
}

bool ssss_combine(const bignum256 *shares, int n, bignum256 *secret)
{
	return true;
}

File: mocks/trezor/crypto/ssss.py

# ../extmod/modtrezorcrypto/modtrezorcrypto-ssss.h
def split(m: int, n: int, secret: bytes) -> tuple:
    '''
    Split secret to (M of N) shares using Shamir's Secret Sharing Scheme
    '''

# ../extmod/modtrezorcrypto/modtrezorcrypto-ssss.h
def combine(shares: tuple) -> bytes:
    '''
    Combine M shares of Shamir's Secret Sharing Scheme into secret
    '''
